docs/community-operators-troubleshooting.md

```diff
@@ -0,0 +1,219 @@
+
+## <a id="apply-test-waivers"></a>apply-test-waivers
+Failures at this step are uncommon. If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="content-hash"></a>content-hash
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="certification-project-check"></a>certification-project-check
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support. 
+
+## <a id="get-organization"></a>get-organization
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-pyxis-certification-data"></a>get-pyxis-certification-data
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="static-tests"></a>static-tests
+The summary of failed static tests will be posted to the Pull Request as a comment with detailed reasons. Fix all of 
+failed static tests and retrigger the hosted pipeline. To learn more about static tests, please follow this [documentation](https://redhat-openshift-ecosystem.github.io/operator-pipelines/users/static_checks/).
+
+## <a id="static-tests-results"></a>static-tests-results
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="merge-registry-credentials"></a>merge-registry-credentials
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="digest-pinning"></a>digest-pinning
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="verify-pinned-digest"></a>verify-pinned-digest
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="dockerfile-creation"></a>dockerfile-creation
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="build-bundle"></a>build-bundle
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="make-bundle-repo-public"></a>make-bundle-repo-public
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="get-supported-versions"></a>get-supported-versions
+Failures at this step are uncommon.  If you do experience a failure or error at this step, contact Red Hat Support.
+
+## <a id="add-bundle-to-index"></a>add-bundle-to-index
+Failures at this step are uncommon and if they do occur they are often transient.  
+
+Please click the `Close pull request` button in GitHub then click the `Reopen pull request` button.
```

Comment:
This is an old way how to restart a pipeline. Instead, we should tell a user how to restart a pipeline using comments.